vernac/comFixpoint.ml

```diff
@@ -247,6 +247,95 @@ let interp_fixpoint ~cofix l ntns =
   check_recursive true env evd fix;
   (fix,pl,Evd.evar_universe_context evd,info)
 
+(** Lifting the uniform arguments of a fixpoint out. *)
+let lift_fixpoints = ref true
+let _ =
+  let open Goptions in
+  declare_bool_option
+    { optdepr  = false;
+      optname  = "";
+      optkey   = ["Fixpoints";"Lift";"Arguments"];
```

Comment:
why "FixpointS"?

Reply:
`Fixpoint` is a keyword.

Reply:
I think I remember a way was found to create a `Set Inductive Cumulativity` option nonetheless.

Reply:
>I think I remember a way was found to create a Set Inductive Cumulativity option nonetheless.

I remember talk of that but in practice we turned `Inductive` into a regular ident in 7b323421ba558011c304a686c4cd368e1ff39440